### ISSUE TITLE
UNSTABLE pull / linux-focal-py3_8-clang9-xla / test (xla)

### ISSUE THREAD

**Issue by clee2000** [pytorch/pytorch#113251]
XLA is a bit flaky at the moment due to test_xla_backend_intf sometimes failing.

Ex https://hud.pytorch.org/pytorch/pytorch/commit/1f3fa13f0ace035f453651c6da3e96cb64413674

I have asked @JackCaoG to skip in XLA repo.  This should be closed after that PR lands and the pin is updated.

cc @seemethere @malfet @pytorch/pytorch-dev-infra

**Comment by JackCaoG** [comment]
Thanks, let me disable he flaky test now

**Comment by JackCaoG** [comment]
I submitted https://github.com/pytorch/xla/pull/5779, I will merge it once CI is done and then I need to update the XLA pin in pytorch so test will be skipped upstream.

**Comment by clee2000** [comment]
It looks to me that xla is stable again after the pin got updated, so I will be closing this. 
If it turns flaky again, feel free to reopen

**Comment by clee2000** [comment]
Fixed after #117999

**Comment by huydhn** [comment]
Reopen this to unblock viable/strict update as the attempt to fix XLA has been reverted https://github.com/pytorch/pytorch/pull/121501.

**Comment by JackCaoG** [comment]
new attempt pr in https://github.com/pytorch/pytorch/pull/121529, xla ci is in progress

**Comment by huydhn** [comment]
Trunk has recovered, so let's close this.

**Comment by huydhn** [comment]
Another issue showing up, one XLA test is failing after https://github.com/pytorch/pytorch/pull/120632 while the job was marked as unstable during the streak of XLA trunk failures earlier today.  It looks like a change on XLA is needed, so I'm reopening this until that change lands.

**Comment by albanD** [comment]
The xla failure in this PR is an NPM issue, I don't think it is from the PR itself?
That PR did require an xla update which was merged at https://github.com/pytorch/xla/pull/6625 and the PT pin was updated past that xla PR before the pytorch side PR was merged.

**Comment by huydhn** [comment]
Closing this as the fix https://github.com/pytorch/pytorch/pull/121529 has been landed and trunk looks green now

**Comment by clee2000** [comment]
Re opening this due to the the failure on https://hud.pytorch.org/pytorch/pytorch/commit/55f36d1ada09917195892443295baf1cd0e5cfdd https://github.com/pytorch/pytorch/actions/runs/8429876651/job/23086500409.  Need some time to figure when it started as none of the PRs seem particularly relevant

**Comment by clee2000** [comment]
The above PR got reverted but xla was still failing.  Merged a pin update https://hud.pytorch.org/pytorch/pytorch/commit/04399a30913fd04c2120420b671cd432659d56e6 and it's is now green
